synth.py

```diff
@@ -186,10 +186,10 @@ def _feed_thread(self):
 
 
 @contextmanager
-def open_sc_stream(samplerate=SAMPLERATE, buffer_duration=1.0):
+def open_sc_stream(samplerate=SAMPLERATE, buffer_duration=1):
```

Comment:
```suggestion
def open_sc_stream(samplerate=SAMPLERATE, buffer_duration=1.0):
```
I would leave this as a float number to signify this is a float (or we can just start using type annotations)